### PR TITLE
`RecordProperties` do not create Logs

### PR DESCRIPTION
**Pull request by evantahler**
This PR removes Logging from RecordProperties.  This was creating a large volume of logs on every import.  These logs are not required as we store various timestamps on each RecordProperty to share the value's freshness.  This also removed a place in which PII was stored, as we were previously storing old and new property values in the log messages.  

In our demo dataset importing 1000 records:
* Before this PR: 26,247 Log Rows
* After this PR: 4,282 Log rows 

... that's ~85% less logs!

### Development

- [x] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

 * This PR should /improve/ database performance! 

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: core/src/models/GrouparooModel.ts [hunk 1]
```diff
@@ -90,13 +90,6 @@ export class GrouparooModel extends LoggedModel<GrouparooModel> {
     return instance;
   }
 
-  @BeforeCreate
-  static generateId(instance: GrouparooModel) {
-    if (!instance.id) {
-      instance.id = `${instance.idPrefix()}_${uuid.v4()}`;
-    }
-  }
```
Comment (evantahler) [inline core/src/models/GrouparooModel.ts:98]: This was redundant with the method within `LoggedModel` that GrouparooModel extends

File: core/src/models/Import.ts [hunk 2]
```diff
@@ -230,9 +230,7 @@ export class Import extends Model {
 
   @BeforeCreate
   static generateId(instance: Import) {
-    if (!instance.id) {
-      instance.id = `${instance.idPrefix()}_${uuid.v4()}`;
-    }
+    if (!instance.id) instance.id = `${instance.idPrefix()}_${uuid.v4()}`;
```
Comment (evantahler) [inline core/src/models/Import.ts]: Cleaning up other instances that don't extend LoggedModel too

File: core/src/models/RecordProperty.ts [hunk 3]
```diff
@@ -25,7 +30,10 @@ const STATE_TRANSITIONS = [
 ];
 
 @Table({ tableName: "recordProperties", paranoid: false })
-export class RecordProperty extends LoggedModel<RecordProperty> {
+export class RecordProperty extends Model {
+  @Column({ primaryKey: true })
+  id: string;
```
Comment (evantahler) [inline core/src/models/RecordProperty.ts]: Brining back id, createdAt, and updatedAt that we previously inherited from LoggedModel

File: core/src/models/Export.ts [hunk 4]
```diff
@@ -293,10 +293,8 @@ export class Export extends Model {
   }
 
   @BeforeCreate
-  static generateId(instance) {
-    if (!instance.id) {
-      instance.id = `${instance.idPrefix()}_${uuid.v4()}`;
-    }
+  static generateId(instance: Export) {
+    if (!instance.id) instance.id = `${instance.idPrefix()}_${uuid.v4()}`;
```
Comment (bleonard) [inline core/src/models/Export.ts]: Why do these keep happening. Some sort of linter?

File: core/src/models/RecordProperty.ts [hunk 5]
```diff
@@ -25,7 +30,10 @@ const STATE_TRANSITIONS = [
 ];
 
 @Table({ tableName: "recordProperties", paranoid: false })
-export class RecordProperty extends LoggedModel<RecordProperty> {
```
Comment (bleonard) [inline core/src/models/RecordProperty.ts:28]: Not 100% sure we'll be happy with the decision if `RecordProperty` should share any behaviors now or in the future with all the other ones. Maybe `LoggedModel` should really be called `CommonModel` and logging is turned off. Something to keep an eye on.

File: core/src/models/Export.ts (hunk 4 above)
Comment (bleonard) [reply core/src/models/Export.ts]: fwiw and i don't feel that strongly about it, but I think the braces prevent mistakes even though one line looks nicer.

File: core/src/models/RecordProperty.ts (hunk 5 above)
Comment (bleonard) [reply core/src/models/RecordProperty.ts:28]: For example, id generation, timestamps, etc

**Review by bleonard — APPROVED** [review]

File: core/src/models/Export.ts (hunk 4 above)
Comment (evantahler) [reply core/src/models/Export.ts]: Nah, it's me manually doing it with a find/replace when I find some duplicated code to clean up.  Prettier seems to be OK with it, and I find them much easier to read.

File: core/src/models/RecordProperty.ts (hunk 5 above)
Comment (evantahler) [reply core/src/models/RecordProperty.ts:28]: Yeah - we've got ~6 models now that don't extend Logged Model.  I like that though - I'll make a chain `CommonModel -> LoggedModel -> ActualModel`

**Comment by evantahler** [comment]
Now all models extend `CommonModel<T>`, which includes the id and timestamp columns, as well as some utilities around formatting IDs.

**Pull request merged**